### ISSUE TITLE
[Feature Request] Addition of new NEAT machines and  other new PCem machines.

### ISSUE THREAD

**Issue by Fenix770** [86Box/86Box#469]
Hello. 
I write to ask if it's possible to 

A. add some of the new Pcem machines, (like, Tulip  286, Packarbell Legend 300SX and the AMI 286 bios, which uses a Pc Chips M-209 chip, maybe both AMI 286 machines can cohexist with different names). PD: The real machine has an Addtech chipset which  provides EMS memory, but has not been implemented and also I don´t know if documentation exists.  

B. Add some new NEAT 286 machines (if NEAT emulation is complete) Because the NEAT AMI 286 bios is really difficult for me to configure, (and I don´t know if, either I configured wrong the chipset or the EMS emulation does not work or is not complete). 
The newer NEAT 286 bios may have an easier configuration like those which has SCAT bios (I downloaded some of the from the Vogons site). I wrote a question about how to configure the Neat chipset in Varcem, but I haven´t any response so far. 

PD: There is no Neat chipset emulation in Pcem

Thanks in advance!

**Comment by OBattler** [comment]
Actually, PCem does have NEAT emulation, but it's very rudimentary and incomplete. I'll look into adding NEAT machine, it shouldn't be that difficult.

As for the other machines emulated, any machine implemented by Dns2k would need a lot of work done since he just shoves the BIOS in and is happy if he sees it POST, and usually never implement the actual chipset.

**Comment by Fenix770** [comment]
The AMI 286 bios with Pc chips M-209 were one of the first machines emulated with generic XT bios
As for the  Packardbell Legend 300  I tested it with Pcem v15 with 386 SX and Cyrix 486 SLC and Windows 3.1 installed  it seems to work right.  
With  Tulip 286  it requires to be configured with the setup disk (like the IBM machines) but is only in Ducth language, worked when configured correctly. But I only installed DOS so I don´t know for sure if the machine emulation is complete.
